data/helpers.d/getopts

```diff
@@ -152,6 +152,15 @@ ynh_handle_getopts_args () {
 								fi
 								# Escape double quote to prevent any interpretation during the eval
 								all_args[$i]="${all_args[$i]//\"/\\\"}"
+								# Escape $ as well to prevent the string following it to be seen as a variable.
+								all_args[$i]="${all_args[$i]//$/\\\$}"
+
+								# For the record.
+								# We're using eval here to get the content of the variable stored itself as simple text in $option_var...
+								# Other ways to get that content would be to use either ${!option_var} or declare -g ${option_var}
+								# But... ${!option_var} can't be used as left part of an assignation.
+								# declare -g ${option_var} will create a local variable (despite -g !) and will not be available for the helper itself.
+								# So... Stop fucking arguing each time that eval is evil... Go find an other working solution if you can find one!
```

Comment:
I'm in favour of a disclaimer, but maybe avoid rude words...? :wink:

Reply:
I like things to be clear...
And I'm not usually smooth and politically correct...
However, ~I don't care...~ `[Politically mode ON]` *I'm not feeling absolutely concerned by what should be done with the text of this disclaimer and the way things should be said to be understood.*

Reply:
Sill I find eval is evil :)